foreman/data_refinery_foreman/surveyor/test_end_to_end.py

```diff
@@ -69,10 +159,17 @@ def wait_for_job(job, job_class: type, start_time: datetime, loop_time: int = No
 # job in the database because it'd be stuck in a transaction.
 class NoOpEndToEndTestCase(TransactionTestCase):
     @tag("slow")
-    def test_no_op(self):
+    @vcr.use_cassette(
+        "/home/user/data_store/cassettes/surveyor.test_end_to_end.no_op.yaml",
+        ignore_hosts=["nomad"],
+    )
+    @patch("data_refinery_foreman.surveyor.surveyor.ArrayExpressSurveyor")
```

Comment:
What about having an additional decorator that does the `@patch` with the side effects below? It would be nice if we could do `@patch_surveyor("ARRAY_EXPRESS")` at the top of the tests. We can also explore this down the road.

Reply:
Mmm yeah, that's not a bad idea but I think it could get pretty complicated because it'd be a wrapper around a wrapper. I'd rather not shoehorn it into this PR.